v4/source/users.yaml

```diff
@@ -1118,3 +1118,25 @@
         '501':
           $ref: '#/responses/NotImplemented'
 
+  /users/stats:
+    get:
+      tags:
+        - users
+      summary: Get users stats
+      description: |
+        Get users stats on the system.
```

Comment:
Could you update this to say:
```
Get statistics for users on the system.

**Minimum server version:** unreleased
```

Once the code gets merged in we can update the minimum server version to be whatever the next one is